File: Front/src/app/model/despesa.ts
export class Despesa {
    codigo?: number;
    nome ?: string;
    valor ?: number;
    descricao ?: string;
    dtCriacao ?: string;
    dtQuitada ?: string;
    origem ?: string;
    codigoOrigem?: number;
    ativo ?: boolean;
    parte ?: number;
    constructor(codigo?: number, nome ?: string, valor ?: number, descricao ?: string, ativo ?: boolean, origem ?: string, codigoOrigem?: number) {
      this.codigo = codigo;
      this.nome = nome;
      this.valor = valor;
      this.descricao = descricao;
      this.ativo = ativo;
      this.origem = origem;
      this.codigoOrigem = codigoOrigem;
    }
}

File: wowsca/src/main/java/jwt/validation/wowsca/model/Despesa.java
package jwt.validation.wowsca.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Despesa {
    private int codigo;
    private String nome;
    private String descricao;
    private double valor;
    private String dtCriacao;
    private String dtQuitada;
    private boolean ativo;
    private String origem;
    private int codigoOrigem;
    
}
